recipes/imagehash/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "3.1" %}
+
+package:
+  name: imagehash
+  version: {{ version }}
+
+source:
+  fn: imagehash-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/0c/1c/b6f539cc08cf8cfb12cb59a6883ee7f1cc0937cf3709247c2a15c8a1e665/ImageHash-{{ version }}.tar.gz
+  md5: 0bf909b6d138768f80dd689be7bedaaa
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy x.x
```

Comment:
I am not sure it needs `numpy x.x`. I could not find `numpy.get_include()` anywhere.